### ISSUE TITLE
Tensorflow 2.1: How does the metric 'tf.keras.metrics.PrecisionAtRecall' works with multiclass-classification? 

### ISSUE THREAD

**Issue by FrancescaAlf** [tensorflow/addons#1211]
I am wondering how this metrics works in case of multiclass classification. Does it compute the average between values of precision belonging to each class? 

I have a multiclass-classification problem, with three classes. I am interested in calculate the PrecisionAtRecall when the recall value is equal to 0.76, only for a specific class . What I have done was just setting 0.76 in brackets : 
'' model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy', tf.keras.metrics.PrecisionAtRecall(0.76)], sample_weight_mode='temporal') ''
But I am sure is not the right way. 
How can I adjust my code to reach my goal?
Thanks

**Comment by gabrieldemarmiesse** [comment]
Thanks for your question. This doesn't not concern tensorflow addons. Tensorflow addons is imported with `import tensorflow_addons`. For matters related to tf.keras, see the tensorflow/tensorflow repository.
